geode-core/src/main/java/org/apache/geode/management/internal/configuration/validators/MemberValidator.java

```diff
@@ -64,12 +61,13 @@ public void validateCreate(CacheElement config, ConfigurationManager manager) {
     }
   }
 
-  public String[] findGroupsWithThisElement(CacheElement config, ConfigurationManager manager) {
+  public String[] findGroupsWithThisElement(RestfulEndpoint<?> config,
+      ConfigurationManager manager) {
     // if the same element exists in some groups already, make sure the groups has no common members
     List<String> groupWithThisElement = new ArrayList<>();
     for (String group : persistenceService.getGroups()) {
       CacheConfig cacheConfig = persistenceService.getCacheConfig(group);
-      if (cacheConfig != null && manager.get(config.getId(), cacheConfig) != null) {
+      if (cacheConfig != null && manager.has(config.getId(), cacheConfig)) {
```

Comment:
because `get` now does an implicit `list` in order to populate runtime properties, it's a bit heavyweight for simple validators' needs, so I added a lightweight API if you just need to check if already exists in the current configuration.

Reply:
reverted this bit :)